CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ### Unreleased
-  - Fix incorrect index date time
+  - Move transformers to be memory based instead of file based
```

Comment:
As an FYI, i think when we release this it should be version 3.0.0

Reply:
copied the style from some existing classes , will remove